keras_nlp/src/tests/test_case.py

```diff
@@ -515,6 +523,13 @@ def run_vision_backbone_test(
                 run_mixed_precision_check=run_mixed_precision_check,
                 run_quantization_check=run_quantization_check,
             )
+            if expected_pyramid_output_keys:
```

Comment:
let's just run this once above, doubling up might inflate our test sizes too much (a perennial problem)